R/Rules-class.R

```diff
@@ -1519,7 +1544,11 @@ StoppingCohortsNearDose <- function(nCohorts = 2L,
 #' @note Typically, end users will not use the `.DefaultStoppingCohortsNearDose()` function.
 #' @export
 .DefaultStoppingCohortsNearDose <- function() { # nolint
-  StoppingCohortsNearDose(nCohorts = 3L, percentage = 0.2)
+  StoppingCohortsNearDose(
+    nCohorts = 3L,
+    percentage = 0.2,
+    report_label = NA_character_
```

Comment:
```suggestion
    percentage = 0.2
```
not needed explicitly here, it is the default for that argument anyway. similar below for other default constructors